pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -550,6 +552,29 @@ var _ = Describe("Workload Updater", func() {
 			controller.Execute()
 			testutils.ExpectEvent(recorder, FailedChangeAbortionReason)
 		})
+
+		It("shouldn't cancel the migration if the migration object is still in running phase but the domain is ready on the target", func() {
```

Comment:
This doesn't seem to reproduce the problem.

Reply:
Isn't the flakes happening when the state of the migration isn't finalized by the actual migration has already finished?

Reply:
@alicefr I tried to run this test alone without the fix and this did not fail for me. What I am trying to say is that I expect it to fail.

Reply:
You were right, the unit test was wrong, I add the change condition while the VMI shouldn't have it anymoe